src/Servers/IIS/IIS/test/Common.FunctionalTests/GlobalVersionTests.cs

```diff
@@ -255,12 +255,12 @@ private static void CopyShimToOutput(IISDeploymentParameters parameters)
 
     private static void CopyFiles(DirectoryInfo source, DirectoryInfo target)
     {
-        foreach (var directoryInfo in source.GetDirectories())
+        foreach (DirectoryInfo directoryInfo in source.GetDirectories())
```

Comment:
bit of nice cleanup we're losing. anything important @HaoK❔

Reply:
Nah just minor stuff, hoping to revert this revert someday anyways